src/lib/polymer-bootstrap.html

```diff
@@ -47,6 +49,22 @@
       return factory;
     };
 
+    var flatten = function(child) {
```

Comment:
This feels super weird. If we want to attach the Polymer.Base prototype to the right spot (just above HTMLElement), then the proper procedure would be to write a function that traverses the given prototype chain and attaches Polymer.Base in the correct spot.